static/js/components/CardGrid.js

```diff
@@ -0,0 +1,57 @@
+import React, { Component } from "react";
+import CardComponent from "./CardComponent"
+
+require('../../css/CardComponent.scss');
+
+export default class CardGrid extends Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {};
+    }
+
+    componentDidMount() {
+        fetch('http://localhost:5000/api/items/all')
```

Comment:
this should be abstracted from localhost to the current base url

Reply:
Yes, that was an oversight

Reply:
will make an issue about this